Sources/ScipioKit/BuildOptions.swift

```diff
@@ -33,6 +72,10 @@ struct BuildOptions: Hashable, Codable, Sendable {
     /// - Note: It have to store the actual file contents rather than its path,
     /// because the cache key should change when the file contents change.
     let customFrameworkModuleMapContents: Data?
+
+    #if compiler(>=6.0)
+    private(set) var environment: Environment?
```

Comment:
How about renaming this like `swiftPMEnvironment`? I feel `Environment` seems to be too generic a name.

Reply:
`toolchainEnvironment` seems to be a suitable name!